docs/developers/building.md

```diff
@@ -42,18 +42,11 @@ And build using the build script:
     These instructions were tested with Ubuntu 18.04. Depending on the
     Linux distribution being used, these instructions may vary.
 
-First enable `deb-src` via:
-
-    sudo gedit /etc/apt/sources.list
-
-In this file, uncomment first `deb-src` line. Now refresh packages:
-
-    sudo apt-get update
-
 Install build deps:
 
-    sudo apt-get build-dep qemu
-    sudo apt-get install git libsdl2-dev libepoxy-dev
+    sudo apt-get update
+    sudo apt-get install git libsdl2-dev libepoxy-dev build-essential \
+    pkg-config zlib1g-dev libsdl2-dev libpixman-1-dev
```

Comment:
Avoid putting `pkg-config` (a binary name)  at the front of the line - maybe also indent? I was wondering why you were running `pkg-config` after installing packages, until I realized it was a package to be installed.

Reply:
I've fixed your concerns here. RE: indentation I'm following the style of the other entries.